bundles/k3d-standard/uds-bundle.yaml

```
@@ -223,3 +223,14 @@ packages:
           values:
             - path: "falcosidekick.replicaCount"
               value: 1
+        uds-falco-config:
+          variables:
+            - name: FALCO_SANDBOX_RULES_ENABLED
+              description: Enable sandbox rules
+              path: sandboxRulesEnabled
+            - name: FALCO_INCUBATION_RULES_ENABLED
+              description: Enable incubation rules
+              path: incubationRulesEnabled
```

nit: maybe keep the naming the same as the upstream name (incubating vs incubation)
```suggestion
            - name: FALCO_INCUBATING_RULES_ENABLED
              description: Enable incubating rules
              path: incubatingRulesEnabled
```